sympy/plotting/tests/test_plot.py

```diff
@@ -288,7 +296,7 @@ def test_append_issue_7140():
 
     x = Symbol('x')
     p1 = plot(x)
-    p2 = plot(x**2)
+    p2 = plot(x ** 2)
```

Comment:
Whitespaces around `*`, `/`, and `**` are unnecessary. I think it's better to leave above changes as-is and create a new PR if it needs to be fixed, as it's not part of this PR.